jackal_base/scripts/calibrate_compass

```diff
@@ -2,7 +2,8 @@
 
 TEMPDIR=$(mktemp -d --tmpdir calibrate_compass.XXXX)
 BAG_FILE=${TEMPDIR}/imu_record.bag
-CAL_FILE=${TEMPDIR}/imu_compass.yaml
+CAL_FILE=${TEMPDIR}/mag_config.yaml
+CAL_FINAL_PATH=${ROS_ETC_DIR}/jackal_base
```

Comment:
Not sure if I agree there for the main group of users. Researchers like recalibrating things :)